docs/features/multiple-networks/multi-homing.md

```diff
@@ -67,6 +120,7 @@ spec:
   of the surrounding object.
 
 **NOTE**
```

Comment:
use github's [alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts).